pkg/cmd/server/start/admission.go

```diff
@@ -41,6 +41,7 @@ var (
 
 func init() {
 	defaultOffPlugins.Insert("AlwaysPullImages")
+	defaultOffPlugins.Insert("JenkinsBootstrapper")
```

Comment:
I do not want to remove the enabled flag.  It went in so we can do two things:

1. Default the entire config so an admin can *see* what is used
2. Allow jenkins auto-creation to be disabled from a single spot.

Ansible should create config with the enabled field set to empty, not to false or true.